src/lib/client.ts

```diff
@@ -86,8 +85,10 @@ export class Client {
                   // TODO
                 }
               })
-              const addr: string | null = LocalStorage.getItem("rewardAddress")
-              const addr2: string = addr ?? farmerPublicKey
+              const addr_unchecked: string | null = LocalStorage.getItem("rewardAddress")
+              let addr!: string;
```

Comment:
You're just shutting up TypeScript compiler while showing at the same time that you're doing it incorrectly because `addr_unchecked` clearly may or may not be there, so `addr` is not guaranteed to be `string`.